src/app/util/web3.service.ts

```diff
@@ -32,7 +32,7 @@ export class Web3Service {
       // Hack to provide backwards compatibility for Truffle, which uses web3js 0.20.x
       Web3.providers.HttpProvider.prototype.sendAsync = Web3.providers.HttpProvider.prototype.send;
       // fallback - use your fallback strategy (local node / hosted node + in-dapp id mgmt / fail)
-      this.web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'));
+      this.web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:7545'));
```

Comment:
Things like the item changed here (the port number for localhost) should probably be kept in a configuration file which is not tracked by source control. That way, each person can keep their own system settings without everyone having to use the same thing on their dev environments. I've used this package in the past to handle my configuration options: https://www.npmjs.com/package/dotenv